benchmarks/edge_profiles.py

```diff
@@ -17,18 +17,18 @@ def fill_ggd(edge_profiles, times):
     """Fill nested arrays of structures in grids_ggd and ggd substructures.
 
     Args:
-        edge_profiles: edge_profiles IDS object (either from IMASPy or AL HLI)
+        edge_profiles: edge_profiles IDS object (either from imas-python or AL HLI)
         times: time values to fill
     """
     edge_profiles.ids_properties.homogeneous_time = (
-        imaspy.ids_defs.IDS_TIME_MODE_HETEROGENEOUS
+        imas.ids_defs.IDS_TIME_MODE_HETEROGENEOUS
     )
-    edge_profiles.ids_properties.comment = "Generated for IMASPy benchmark suite"
+    edge_profiles.ids_properties.comment = "Generated for imas-python benchmark suite"
```

Comment:
Here and for similar cases below.
```suggestion
    edge_profiles.ids_properties.comment = "Generated for IMAS-Python benchmark suite"
```

Reply:
Updated in https://github.com/iterorganization/IMAS-Python/tree/rename_imaspy_sources_to_imas